app/javascript/mastodon/features/home_timeline/components/explore_prompt.jsx

```diff
@@ -15,9 +15,11 @@ export const ExplorePrompt = () => (
     <h1><FormattedMessage id='home.explore_prompt.title' defaultMessage='This is your home base within Mastodon.' /></h1>
     <p><FormattedMessage id='home.explore_prompt.body' defaultMessage="Your home feed will have a mix of posts from the hashtags you've chosen to follow, the people you've chosen to follow, and the posts they boost. It's looking pretty quiet right now, so how about:" /></p>
 
-    <div className='dismissable-banner__message__actions'>
-      <Link to='/explore' className='button'><FormattedMessage id='home.actions.go_to_explore' defaultMessage="See what's trending" /></Link>
-      <Link to='/explore/suggestions' className='button button-tertiary'><FormattedMessage id='home.actions.go_to_suggestions' defaultMessage='Find people to follow' /></Link>
+    <div className='dismissable-banner__message__wrapper'>
```

Comment:
This is inconsistent with the CSS class name.

```suggestion
    <div className='dismissable-banner__message__actions__wrapper'>
```